src/operator/tensor/sample_op.cu

```diff
@@ -67,10 +67,10 @@ void SampleNormal_<gpu>(const nnvm::NodeAttrs& attrs,
   }
 }
 
-NNVM_REGISTER_OP(uniform)
+NNVM_REGISTER_OP(random_uniform)
```

Comment:
this was failing on my machine but seems was fine on test server. I fixed it to be safe.